README.md

```diff
@@ -118,6 +118,8 @@ There are additional attributes that allow you to customize each tour-tip.
 
 * `tourtip-container-element` **(Default: "body")**: Element the tour tips are placed in. Especially practical when you have several scrollable containers you want to display tours in.
 
+* `tourtip-title` **(Default: null)**: Title creates a separate h2 element to allow the separation of a title and it's content.
```

Comment:
minor, but documentation indicates h2 but the template is actually h3. mind amending / flattening your commit with that tweak?